spring-cloud-gcp-samples/pom.xml

```diff
@@ -166,7 +166,7 @@
 								<gcs-local-directory>/tmp/gcp_integration_tests/integration_storage_sample</gcs-local-directory>
 								<spring.cloud.gcp.sql.instance-connection-name>spring-cloud-gcp-ci:us-central1:testmysql</spring.cloud.gcp.sql.instance-connection-name>
 								<spring.cloud.gcp.sql.database-name>code_samples_test_db</spring.cloud.gcp.sql.database-name>
-								<spring.datasource.password>test</spring.datasource.password>
+								<spring.datasource.password>${db.root.password}</spring.datasource.password>
```

Comment:
If you define `--define "spring.datasource.password=${DB_PASSWORD}"`, do you still need this line at all?

Reply:
Ah, true it would not be necessary. I just changed it